### PR TITLE
Fix #6404 : Version dropdown on download page doesn't scroll on overflow

### LINKED ISSUE #6404: Version dropdown on download page doesn't scroll on overflow
### URL:

https://beta-node-js-org.vercel.app/en/download

### Browser Name:

Chrome

### Browser Version:

122.0.6261.94

### Operating System:

macOS 14.3.1

### How to reproduce the issue:

1. Went to the new beta Node.js website's download page https://beta-node-js-org.vercel.app/en/download
2. Clicked the version dropdown (also known as a combobox or select)
3. Expected it to be scrollable if the complete list cannot be shown within the visible viewport
4. Instead, the overflow is hidden and thus means I cannot see the full list unless I zoom out.

image

### PR DESCRIPTION
**Pull request by MohitKambli**
## Description
Hello,
Hope you're doing well.
This PR has been raised to fix issue #6404, wherein the scroll down feature for versions doesn't work as it actually should.
In order to provide a fix for the same, I have added style property to the **_Primitive.Content_** component for which **_maxHeight_** and **_overflowY_** properties have been defined in the **_index.tsx_** file located at _**components/Common/Select**_.

## Validation
In order to confirm if the scroll feature works as per requirement, I started the server locally and checked if the desired behaviour has been achieved or not.
Following are the screenshots of the updated dropdown component:
1) 
image

2)
image

If at all there are any changes required from my end, then do let me know.
Sincere apologies if I made any mistakes while resolving this issue and raising a PR for the same.

## Related Issues
Fixes #6404

### Check List

- [X] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [X] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [X] I have run `npx turbo format` to ensure the code follows the style guide.
- [X] I have run `npx turbo test` to check if all tests are passing.
- [X] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Review by AugustinMauroy — CHANGES_REQUESTED** [review]
Please use css-module and tailwindcss.

**Comment by MohitKambli** [comment]
Hello @AugustinMauroy,
I have made the necessary changes in the index.module.css file.
If at all there is something else that I need to do, then do let me know.
Sincere apologies for messing up in the previous commit and for my mistakes if you find any in the new commit.
Thank you so so much for understanding my part.

Thanks and Regards,
Mohit Kambli

File: components/Common/Select/index.module.css [hunk 1]
```diff
@@ -59,6 +59,8 @@
 }
 
 .dropdown {
+  @apply max-h-48
+    overflow-y-auto;
   @apply max-w-xs
```
Comment (AugustinMauroy) [inline components/Common/Select/index.module.css]:
```suggestion
  @apply max-h-48
    overflow-y-auto
    max-w-xs
```

You can simplify doing that. Also don't miss to `npx turbo format`

File: components/Common/Select/index.tsx [hunk 2]
```diff
@@ -90,6 +90,7 @@ const Select: FC<SelectProps> = ({
           <Primitive.Content
             position={inline ? 'popper' : 'item-aligned'}
             className={classNames(styles.dropdown, { [styles.inline]: inline })}
+            style={{ maxHeight: '200px', overflowY: 'auto' }}
```
Comment (AugustinMauroy) [inline components/Common/Select/index.tsx]:
```suggestion
```
You can totally remove this line

**Comment by MohitKambli** [comment]
Hello @AugustinMauroy,
As you had mentioned in the comments, I incorporated the code changes.
If at all there is something else required from my end, then do let me know.
Sincere apologies for the trouble being caused.

Thanks and Regards,
Mohit Kambli

**Comment by ovflowd** [comment]
> Hello @AugustinMauroy, As you had mentioned in the comments, I incorporated the code changes. If at all there is something else required from my end, then do let me know. Sincere apologies for the trouble being caused.
> 
> Thanks and Regards, Mohit Kambli

I'd appreciate if you did not use AI/ChatGPT based solutions for generating your comments.

**Comment by MohitKambli** [comment]
> > Hello @AugustinMauroy, As you had mentioned in the comments, I incorporated the code changes. If at all there is something else required from my end, then do let me know. Sincere apologies for the trouble being caused.
> > Thanks and Regards, Mohit Kambli
> 
> I'd appreciate if you did not use AI/ChatGPT based solutions for generating your comments.

Hello @ovflowd,
No, I don't use AI/ChatGPT at all to write my comments.
It has just become a habit of writing something like this after every mistake/commit I contribute to the Open Source Repositories.
That's just how I am.

Thanks and Regards,
Mohit Kambli

**Comment by AugustinMauroy** [comment]
> Sincere apologies for messing up in the previous commit and for my mistakes if you find any in the new commit.

There's nothing to worry about! This is your first contribution, and we have a very strict guiding style which helps maintain the website.

**Review by AugustinMauroy — APPROVED** [review]
LGTM ! Thanks for you first contribution 🎉

**Comment by AugustinMauroy** [comment]
I'm requiring fast-tracking 'cause it's download page is difficult to use without this change.

**Review by aymen94 — APPROVED** [review]

**Pull request merged**